go.mod

```diff
@@ -6,34 +6,71 @@ require (
 	github.com/akutz/gofsutil v0.1.2
 	github.com/akutz/gosync v0.1.0 // indirect
 	github.com/akutz/memconn v0.1.0
-	github.com/container-storage-interface/spec v1.0.0
+	github.com/container-storage-interface/spec v1.1.0
```

Comment:
Can you use `github.com/container-storage-interface/spec v1.0.0`. 
Please update only relevant modules needed for e2e testing in the go mod file.